src/main.ts

```diff
@@ -30,9 +30,16 @@ Vue.use(Vue2Filters) // needed by SbcFeeSummary
 
 // main code
 async function start () {
-  // fetch config from environment and API
-  // must come first as inits below depend on config
-  await fetchConfig()
+  // get config from environment
+  const processEnvBaseUrl: string = process.env.BASE_URL // eg, /business/
+  const windowLocationPathname = window.location.pathname // eg, /business/CP1234567/...
+  const windowLocationOrigin = window.location.origin // eg, http://localhost:8080
+  const applicationUrl = windowLocationOrigin + processEnvBaseUrl
+  await setBaseRouteAndBusinessId(windowLocationPathname, processEnvBaseUrl, windowLocationOrigin) // will throw error
```

Comment:
Haha, no, should be "may throw error" :)

Reply:
Haha, good point!  I hope it doesn't always throw an error.  I'll fix the comment.